models/Reservation.js

```diff
@@ -0,0 +1,28 @@
+import mongoose from 'mongoose';
+const { Schema } = mongoose;
+
+const reservationSchema = new mongoose.Schema({
+  _id: Schema.Types.ObjectId,
+  date: {
+    type: String,
+    required: [true, 'Field date is required'],
+  },
+  hour: {
+    type: String,
+    required: [true, 'Field hour is required'],
+  },
+  customerName: {
+    type: String,
+    required: true,
+  },
+  customerLastName: {
+    type: String,
+    required: [true, 'Field customer last is required'],
```

Comment:
Think about error message, this message would be seen be user, maybe "Customer last name is required" is enough?